website/docs/deployment.mdx

```diff
@@ -99,6 +99,7 @@ GitHub enterprise installations should work in the same manner as github.com; yo
 | Name          | Description                                     |
 | ------------- | ----------------------------------------------- |
 | `GITHUB_HOST` | The domain name of your GitHub enterprise site. |
+| `GITHUB_PORT` | The port of your GitHub enterprise site.        |
```

Comment:
can you also add it to `https://docusaurus.io/docs/docusaurus.config.js#githubhost` ?

Reply:
Yes. Sure. Done.